tox.ini

```diff
@@ -44,7 +44,8 @@ disable=missing-module-docstring,
         duplicate-code,
         too-many-arguments,
         consider-using-with,
-        unspecified-encoding
+        unspecified-encoding,
+	      too-many-locals,
```

Comment:
Alternatively, add `# pylint: disable=too-many-locals` in function where this is the issue.

Reply:
ack